import-export/ImportAsJSONAction.js

```diff
@@ -0,0 +1,200 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2017, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+ 
+define(['zepto'], function ($) {
+	'use strict';
+    
+    var IMPORT_FORM = {
+        name: "Import as JSON",
+        sections: [{
+            name: "Import A File",
+            rows: [{
+                name: 'Select File',
+                key: 'select-file',
+                control: 'import-json',
+                required: true,
+                text: 'Select File'
+            }]
+    	}]
+    };
+
+    function ImportAsJSONAction(exportService, identifierService, 
+        dialogService, openmct, context) {
+        
+        this.openmct = openmct;
+        this.context = context;
+        this.exportService = exportService;
+        this.dialogService = dialogService;
+        this.identifierService = identifierService;
+        this.instantiate = openmct.$injector.get("instantiate");
+    };
+
+    ImportAsJSONAction.prototype.perform = function() {
+        var input;
+        this.dialogService.getUserInput(IMPORT_FORM, {})
```

Comment:
Had a hard time formatting this promise chain to be readable. Would it be better practice to used named functions here for resolve and reject?